### DISCUSSION TITLE
How do I add this separator

### DISCUSSION THREAD

**Discussion by rew1nter** [nvim-lualine/lualine.nvim#1264] [Q&A] [answered]
![image](https://github.com/nvim-lualine/lualine.nvim/assets/64508791/e7e160c6-2b79-402b-b877-0d04fdf96148)

**Comment by frolvanya** [comment, marked as answer]
It's just a \ue0b8 () printed two times with different colors
First triangle - fg is blue and bg is gray
Second triangle - fg is gray and bg is dark gray
